src/txmempool.cpp

```diff
@@ -359,7 +359,7 @@ CTxMemPool::CTxMemPool(const CFeeRate& _minReasonableRelayFee) :
     // of transactions in the pool
     nCheckFrequency = 0;
 
-    minerPolicyEstimator = new CBlockPolicyEstimator(_minReasonableRelayFee);
+    minerPolicyEstimator = new CBlockPolicyEstimator();
```

Comment:
Should also drop `CTxMemPool::UpdateMinFee()` (no longer used) to make this compilable.

Reply:
https://github.com/PastaPastaPasta/dash/commit/2ce335ef92fc32f049d18fae976139bbc68bde32